processor/metricstransformprocessor/aggregate_label_values_operation.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2020 OpenTelemetry Authors
+//
```

Comment:
Nit: Would recommend naming these files `operation_aggregate_label_values.go` (similar for the others) so they show up next to each other in the file list.